packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -482,24 +482,26 @@ function useMediaUploadQueue() {
           }
 
           if (isTranscodingEnabled) {
-            currentTranscodingItem.current = id;
-
             if (isGifThatNeedsTranscoding) {
+              currentTranscodingItem.current = id;
               convertGifItem(item);
               return;
             }
 
             if (canTranscodeFile(file)) {
               if (trimData) {
+                currentTranscodingItem.current = id;
                 trimVideoItem(item);
                 return;
               }
 
               if (muteVideo) {
+                currentTranscodingItem.current = id;
                 muteVideoItem(item);
                 return;
               }
 
+              currentTranscodingItem.current = id;
```

Comment:
Just having it once here on line 492 would work too, instead of repeating this 3 times.

```suggestion
            if (canTranscodeFile(file)) {
              currentTranscodingItem.current = id;

              if (trimData) {
                trimVideoItem(item);
                return;
              }

              if (muteVideo) {
                muteVideoItem(item);
                return;
              }

```

Reply:
That would not include gif conversion?

Reply:
I find this version much more readable. 

If you want a one line fix. This would work as well. 

```
          currentTranscodingItem.current = null;
          uploadItem(item);
```

Reply:
No, your change for gif conversion above would stay.

This is just for video optimization.

So two lines changed in total.